_i18n/en/install-guides/windows/windows-slipstream.md

```diff
@@ -45,13 +45,7 @@
    - **Do not** patch mods that are not designed for FTL Multiverse or they may conflict (even small UI mods can sometime conflict, seek help on the MV Discord if you have questions about a specific favorite mod of yours compatibility)
    - Example: ![image](https://user-images.githubusercontent.com/1423894/173681032-76e9b056-ede6-42f5-ad5a-ffa0fece384a.png)
    - Click `Patch`
-4. If you have **AMD** or **Intel** graphics, you'll need an extra step before you can launch FTL Multiverse and **do not** launch it from Slipstream
-5. See: [Checking if Hyperspace is installed correctly](#installed-correctly)
-
-##### AMD/Intel GPUs
```

Comment:
We should add the note about certain VEGA GPUs and to please ping us on discord about it maybe?

I'm not sure I want to mention the other override for vega because everyone will try it blindly and then complain why their other random GPU setup crashes